lib/tasks/avalon.rake

```diff
@@ -24,8 +24,17 @@ namespace :avalon do
     `rails generate active_annotations:install`
   end
 
-  desc "Index MasterFiles and subresources to take advantage of SpeedyAF"
+  desc "Index Admin::Collections and MasterFiles and their subresources to take advantage of SpeedyAF"
   task index_for_speed: :environment do
+    Admin::Collection.find_each do |mf|
+      $stderr.print "c["
+      mf.update_index;
+      mf.declared_attached_files.each_pair do |name, file|
```

Comment:
Should we change the argument here from `mf` to something else for consistency? Maybe `c` or `ac`?

Reply:
Good catch.  I was sloppy with the copy/paste.